### PR TITLE
[docs][cloud] yb_db_admin role

### PR DESCRIPTION
**Pull request by ddhodge**
Details on the yb_db_admin role used to grant yb_superusers aome superuser abilities

@netlify /preview/yugabyte-cloud/cloud-secure-clusters/cloud-users/#yb-db-admin

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md [hunk 1]
```diff
@@ -44,6 +46,7 @@ The following table describes the default YSQL roles and users in YugabyteDB Man
 | --- | --- |
 | [admin](#admin-and-yb-superuser) | The default user for your cluster. If you added your own credentials during cluster creation, the user name will be the one you entered. Although not a superuser, this role is a member of yb_superuser, and you can use it to perform database operations, create other yb_superuser users, create extensions, and manage your cluster. |
 | postgres | Superuser role created during database creation. Not available to YugabyteDB Managed users. |
+| [yb_db_admin](#yb-db-admin) | Role that allows non-superuser users to create tablespaces and perform other privileged operations. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md:49]: Missing description for the role `yb_fdw`.

**Review by aishwarya24 — APPROVED** [review]
LGTM with one comment.

File: docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md [hunk 2]
```diff
@@ -56,10 +60,22 @@ Although not a superuser, `yb_superuser` includes sufficient privileges to perfo
...
 
+### yb_db_admin
+
+The `yb_db_admin` role is specific to YugabyteDB Managed clusters and allows non-superuser roles to perform a number of privileged operations. A user granted this role can do the following:
+
+- alter and drop schemas it does not own
+- alter and drop tables it does not own
+- alter and drop indexes it does not own
+- create tablespaces on tables and indexes it does not own, and drop tablespaces it does not own
+- alter roles it does not own
```
Comment (paullee-yb) [inline docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md:75]: `yb_db_admin` can be thought of a role that passes on superuser privileges. The "it does not own" part places undue emphasis on being able to merely run commands on resources it does not own. It would be better to reword this to more clearly convey the idea that `yb_db_admin` allows users to perform privileged operations much like a superuser (just as a superuser would), of which running operations on objects it does not own being one.

FYI, regarding altering roles, yb_db_admin can only alter the BYPASSRLS attribute of a role. Other things such as giving a role superuser attribute does not work as this would lead to security issues.

File: docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md [hunk 3]
```diff
@@ -56,10 +60,16 @@ Although not a superuser, `yb_superuser` includes sufficient privileges to perfo
...
 - Has the following role options: `INHERIT`, `CREATEROLE`, `CREATEDB`, and `BYPASSRLS`.
 
-- Member of the following roles: `pg_read_all_stats`, `pg_signal_backend`, and [yb_extension](#yb-extension).
+- Member of the following roles: `pg_read_all_stats`, `pg_signal_backend`, `yb_db_admin`, and [yb_extension](#yb-extension).
 
 `yb_superuser` is the highest privileged role you have access to in YugabyteDB Managed. You can't delete, change the passwords, or login using the `postgres` or `yugabyte` superuser roles.
 
+### yb_db_admin
+
+The `yb_db_admin` role is specific to YugabyteDB Managed clusters and allows non-superuser roles to perform a number of privileged operations in a manner similar to a superuser, such as performing operations on objects it does not own. A user granted this role can only alter the BYPASSRLS attribute of a role.
```
Comment (paullee-yb) [inline docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md:69]: "A user granted this role can only alter the BYPASSRLS attribute of a role."

Perhaps we can leave this sentence out for now. We'll add on more role properties yb_db_admin can edit, but it's a work in progress. I'll follow up with you when we have all the role editing properties sorted out (along with updating the functional doc for yb_db_admin).
